### ISSUE TITLE
I want to extract data (myself) from ICES systems in a standard way that does not involve ad hoc changes to the extraction script

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-taf/conjoin#9]
the output view needs to be based on a data wareshouse version of DOME as some of the extraction mappings are quite complex

**Comment by HansMJ** [comment]
Is this not the same issue as [https://github.com/ices-taf/conjoin/issues/7](https://github.com/ices-taf/conjoin/issues/7)

**Comment by neil-ices-dk** [comment]
> Is this not the same issue as #7

not quite; one is about the ICES system and having a flexible data 'pull' service whereas #7 is about the R scipting packages and reading things in a standard way (two halves of the same coin i guess)
